src/components/MenuSection/MenuSection.jsx

```diff
@@ -0,0 +1,33 @@
+import styles from './MenuSection.module.css';
+
+import CardList from 'src/components/CardList/CardList.jsx';
+import Button from 'src/components/Button/Button.jsx';
+
+const MenuSection = () => {
+    const handleButtonClick = (category) => {
+        console.log(`${category} button clicked`);
+    };
+
+    return (
+        <section className={styles.menuSection}>
+            <h1 className={styles.title}>Browse our menu</h1>
+            <p className={styles.description}>
+                <span>Use our menu to place an order online, or </span>
+                <a href="tel:+12345678901" className={styles.phoneTooltip}>
+                    phone
+                    <span className={styles.tooltipText}>+1 (234) 567-8901</span>
+                </a>
+                <span> our store to place a pickup order. Fast and fresh food.</span>
+            </p>
+            <div className={styles.buttonRow}>
+                <Button label="Dessert" onClick={() => handleButtonClick('Dessert')} />
```

Comment:
You could get the categories list from your product list getting uniq values from them for category

Reply:
Done